src/olympia/lib/git.py

```diff
@@ -59,6 +62,74 @@
 SIMILARITY_THRESHOLD = 50
 
 
+# Sometime mimetypes get changed in libmagic so this is a (hopefully short)
+# list of mappings from old -> new types so that we stay compatible
+# with versions out there in the wild.
+MIMETYPE_COMPAT_MAPPING = {
+    # https://github.com/file/file/commit/cee2b49c
+    'application/xml': 'text/xml',
+    # Special case, for empty text files libmime reports
+    # application/x-empty for empty plain text files
+    # So, let's normalize this.
+    'application/x-empty': 'text/plain',
+    # See: https://github.com/mozilla/addons-server/issues/11382
+    'image/svg': 'image/svg+xml',
+    # See: https://github.com/mozilla/addons-server/issues/11383
+    'image/x-ms-bmp': 'image/bmp',
+    # See: https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types#textjavascript  # noqa
+    'application/javascript': 'text/javascript',
+}
+
+
+# Some official mimetypes belong to the `text` category, even though their
+# names don't include `text/`.
+MIMETYPE_CATEGORY_MAPPING = {
+    'application/json': 'text',
+    'application/xml': 'text',
```

Comment:
I know you only moved that but I suspect you'll need `application/xhtml+xml`, `application/rss+xml`, `application/atom+xml` etc too

Reply:
I added https://github.com/mozilla/addons-server/issues/11731 to not blow up the patch even more.

Reply:
Actually… after investigating this in #11731 I figured that `libmagic` doesn't know about these specific types.

XHTML is simply returned as `text/html` and RSS/Atom is returned as `text/xml` which is perfectly fine too.